### PR TITLE
[Bugfix & docs] No Messaging When No Organizations are Created , documentation for `select_organization.dart`, `organization_list.dart` closes #1612

### LINKED ISSUE #1612: No Messaging When No Organizations are Created
**Describe the bug**
1. When no organizations are created, Talawa presents an empty screen in the select organization screen.
1. No information is provided as to how the user must proceed.

**To Reproduce**

1. Install Talawa, Talawa-API and Talawa-Admin as per their installation guides
2. When you try to select an organization in Talawa, you get no error message about why none are being displayed.

**Expected behavior**
We need a popup message that states "No organizations found Please create one using Talawa Admin"."

**Actual behavior**
See above

**Screenshots**
image

**Additional details**
See above

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by SiddheshKukade**
**What kind of change does this PR introduce?**
Feat: adding error dialog in org list page.

**Issue Number:**

Fixes #1612

**Did you add tests for your changes?**
N/A
**Snapshots/Videos:**

- when no organization is present 

- when organizations are present.

**If relevant, did you update the documentation?**
n/a

**Summary**

[feat: changed button text based on org list length](https://github.com/PalisadoesFoundation/talawa/commit/914c7434ff77ecf1df1146efdf8d9343ad79fac2)
[feat: add error dialog for org list page.](https://github.com/PalisadoesFoundation/talawa/commit/e99243f4ce1d54d742bc67e2ecdac772f48cb72f)

**Does this PR introduce a breaking change?**

No

**Other information**

N/A

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

File: lib/widgets/organization_list.dart [hunk 1]
```diff
@@ -69,8 +69,9 @@ class OrganizationList extends StatelessWidget {
               print(model.organizations.isEmpty);
               if (model.organizations.isEmpty) {
                 navigationServiceLocal.showTalawaErrorDialog(
-                    "No organizations found ! Please contact your admin ",
-                    MessageType.error);
+                  "No organizations found ! Please contact your admin ",
```
Comment (palisadoes) [inline lib/widgets/organization_list.dart:72]: Add multilingual responses.

**Review by palisadoes — CHANGES_REQUESTED** [review]
Add multilingual responses.

**Review by palisadoes — CHANGES_REQUESTED** [review]
Add tests

**Comment by palisadoes** [comment]
Please fix the failing tests. Our test code coverage system will fail if either of these two conditions occur:

1) The overall code coverage drops below the target threshold of the repository
2) Any file in the pull request has code coverage levels below the repository threshold

We do this to ensure the overall reliability of the code base is maintained.

**Review by noman2002 — APPROVED** [review]
Looks good to me, the coverage drop is very minimal. I think we should merge this.

**Review by palisadoes — APPROVED** [review]

File: lib/widgets/organization_list.dart (hunk 1 above)
Comment (SiddheshKukade) [reply lib/widgets/organization_list.dart:72]: @palisadoes I actually scheduled to do this for today. I was busy with college assignments for the past 2 days.
 But now It's merged, so  I'll add the multilingual responses in my future PR 👍🏻

Comment (palisadoes) [reply lib/widgets/organization_list.dart:72]: There is a new issue created to fix this

**Pull request merged**